src/components/clouds/style.ts

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+import { colors } from "src/constants";
+
+export const Container = styled.div`
+margin: 0;
+padding: 0;
+`;
+
+export const Image = styled.img`
+object-fit: cover;
+position: absolute; 
+background-repeat: no-repeat;
+background-size: cover;
```

Comment:
Lets install pretter for correct code formating, or add 2 space's for good code looking